### DISCUSSION TITLE
mysqlclient 2.2.5 not accept ssl=False on mariadb connector

### DISCUSSION THREAD

**Discussion by AlvinF01** [PyMySQL/mysqlclient#734] [Issue report]
### Describe the bug

Recently, I setup local development environment for django and just install the latest mysqlclient 2.2.5

But it always return
`MySQLdb.OperationalError: (2026, 'TLS/SSL error: SSL is required, but the server does not support it')`

No matter I set the ssl_mode = DISABLED or ssl = False

Here is the settings.py
```
DATABASES = {
    'default': {
        'ENGINE': 'django.db.backends.mysql',
        "NAME": "my_db",
        "USER": "root",
        "PASSWORD": "",
        "HOST": "127.0.0.1",
        "PORT": "3306",
        'OPTIONS': {'ssl_mode': 'DISABLED', 'ssl' : False}
    }
}

```
After downgrade to mysqlclient 2.2.4 the connection is working fine.

So, I think 2.2.5 has issue not accepting the ssl=False param on mariadb connector

### Environment

Windows: 11
xampp: 8
MariaDB: 10.4.32
python: 3.9.0
mysqlclient: 2.2.5

### How did you install libmysqlclient libraries?

pip install mysqlclient

### What version of mysqlclient do you use?

2.2.5

### Minimum but complete code to reproduce

```python
# Write Python code here.
import MySQLdb

conn = MySQLdb.connect(host='127.0.0.1', port=3306, user='root', ssl=False)
...
```

### Schema and initial data required to reproduce.

```sql
-- Write SQL here.
-- e.g. CREATE TABLE ...
```

**Comment by methane** [comment]
duplicate of #729.
See [this comment](https://github.com/PyMySQL/mysqlclient/discussions/729#discussioncomment-11035085).
